### PR TITLE
[IMP] calendar: user get confirmation on browser when accept,decline …

### PR DESCRIPTION
**Pull request by hmo-odoo**
…and view invitation from email

Task: https://www.odoo.com/web#id=12763&view_type=form&model=project.task&menu_id=3942&action=327

Pad: https://pad.odoo.com/p/r.eagKQBb3I0IH9dtG

### CODE REVIEW

File: addons/calendar/views/calendar_event_templates.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<template id="invitation_view" name="Invitation View">
+    <link rel="stylesheet" href="/calendar/static/src/css/calendar.css"/>
+    <link rel="stylesheet" href="/web/static/lib/bootstrap/css/bootstrap.css"/>
+    <body class="openerp">
+        <div class="container">
+            <div class="row" style="margin-top:30px;">
```
Comment (hmo-odoo) [inline addons/calendar/views/calendar_event_templates.xml]: do not add custom css style in template

File: addons/calendar/static/src/css/calendar.css [hunk 2]
```diff
@@ -44,28 +31,9 @@
...
-.event_status {
-    border : 1px solid;
-    height : 20px;
-    width : auto;
-    background: #808080;
-    color : #FFFFFF;
-    padding: 5px 10px;
-    width: 400px;
-}
 .cal_inline {
```
Comment (hmo-odoo) [inline addons/calendar/static/src/css/calendar.css]: css class should be start with 'o_'

File: addons/calendar/static/src/css/calendar.css [hunk 3]
```diff
@@ -44,28 +31,9 @@
...
-    padding-right : 10px;
-    font-style : italic;
-    font-size : 17px;
-    vertical-align:bottom;
-}
-.cal_image {
-    height: 30px;
-    width : 100px;
-}
 span.no-wrap {
```
Comment (hmo-odoo) [inline addons/calendar/static/src/css/calendar.css]: can you check that it is exits bootstrap class or not ?

File: addons/calendar/static/src/css/calendar.css (hunk 2 above)
Comment (gan-odoo) [inline addons/calendar/static/src/css/calendar.css]: OK css class name start with 'o_'

File: addons/calendar/views/calendar_event_templates.xml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                        <t t-if="current_attendee['state'] != 'needsAction'">
+                            <i t-attf-class="fa #{'text-success fa-check' if current_attendee['state'] == 'accepted' else 'text-danger fa-close'}"></i>
+                            <span class="text-primary">
+                                <strong t-if="current_attendee['state'] == 'accepted'">Yes I'm going.</strong>
+                                <strong t-if="current_attendee['state'] == 'declined'">No I'm not going.</strong>
+                            </span>
+                        </t>
+                        <div><h3><t t-esc="meeting['event']"/></h3></div>
+                        <div>
+                            <div class="row">
```
Comment (hmo-odoo) [inline addons/calendar/views/calendar_event_templates.xml]: need to use multiple row div ? I think, one row div is enough.

Comment (gan-odoo) [inline addons/calendar/views/calendar_event_templates.xml]: @hmo-odoo  Yes, It possible to using one row div, so i made using one row div.

**Comment by tde-banana-odoo** [comment]
Task canceled.

**Pull request closed without merging**